### ISSUE TITLE
can I  use the local F2D directly, not using urn.

### ISSUE THREAD

**Issue by lengyinpeng** [wallabyway/2DCompare#2]
Hi Michael，
      Leaflet data needs server-side generation. Can I generate it locally?
Thanks!

**Comment by wallabyway** [comment]
Great question!

Yes, you can generate the images locally.  

I wrote two blog posts, about 2D leaflet drawings, hoping that would help someone.  The first blog post covers leaflets and how to generate them locally.  This second blog post, covers the 'diff' part.

Take a look at the first blog post here:
https://forge.autodesk.com/blog/viewing-large-ocrterrain-images-forge-viewer

... and look for the title "2. Converting a large image (an OCR scan)"...
That should help you generate your own local leaflets.  
It has a bash script to help convert, a large TIFF image (ie. OCR scan) into a bunch of jpg's (image tile pyramid) which you can then dump into folder and do this '2D compare' demo.

Note: this comparison code, is only for pixels... not for F2D vector stuff.  That will be a follow up blog post.  Is that what you are more interested in ?  F2D comparison?

**Comment by lengyinpeng** [comment]
Yes, I'm very interested in the comparison of F2D vector stuff , and if you post your follow-up blog ,  I'm glad to read your article.Thank you for answering my question. I will read your first blog  to find some inspiration.

**Comment by zhangjing007** [comment]
Hello, I am also interested in f2d vector comparison. The main reason is because I am using forge direct conversion to get f2d. Do you have any new progress on f2d comparison?
